test/distributed/tensor/test_fake.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# Owner(s): ["oncall: distributed"]
```

Comment:
Does this need to be first line for it to work properly?

Reply:
Well, if it needs to be, y'all should fix the linter lol

Reply:
```
ezyang-mac:pytorch-tmp3 ezyang$ git grep -n Owner\(s  | grep 2: | wc -l
     114
```